docs/source/User-guide/Annotate.rst

```diff
@@ -5,168 +5,161 @@ Annotate
+        annotations_dir="./annotations",
+        task_name="railspace",
+        labels=["no_rail_space", "rail_space"],
+        username="rosie",
+    )
 
-	paths:
-	  your_annotation_set:
-		patch_paths: "./path/to/patches/"
-		parent_paths: "./path/to/parents/"
-		annot_dir: "./path/to/save/annotations"
-	  your_annotation_set_2:
-		patch_paths: "./path/to/patches_2/"
-		parent_paths: "./path/to/parents_2/"
-		annot_dir: "./path/to/save/annotations_2"
+.. note:: You can pass either a pandas DataFrame or the path to a csv file as the ``patch_df`` and ``parent_df`` arguments.
 
-For example, if you want to annotate 'rail_space' (as in `this paper <https://dl.acm.org/doi/10.1145/3557919.3565812>`_) and have been using the recommended/default directory structure, your ``annotation_tasks.yaml`` should look like this:
+In the above examples, the following parameters are also specified:
 
-.. code-block:: yaml
+- ``annotations_dir``: The directory where your annotations will be saved (e.g., ``"./annotations"``).
+- ``task_name``: The specific annotation task you want to perform, in this case ``"railspace"``.
+- ``labels``: A list of labels for the annotation task, such as ``"no_rail_space"`` and ``"rail_space"``.
+- ``username``: Your unique identifier, which can be any string (e.g., ``"rosie"``).
 
-	#EXAMPLE
-	tasks:
-	  rail_space:
-		labels: ["no_rail_space", "rail_space"]
+Other arguments that you may want to be aware of when initializing the ``Annotator`` instance include:
 
-	paths:
-	  set_001:
-		patch_paths: "./patches/patch-*png"
-		parent_paths: "./maps/*png"
-		annot_dir: "./annotations_one_inch"
+- ``show_context``: Whether to show a context image in the annotation interface (default: ``False``).
+- ``sortby``: The name of the column to use to sort the patch Dataframe (e.g. "mean_pixel_R" to sort by red pixel intensities).
+- ``delimiter``: The delimiter to use when reading your data files (default: ``","`` for csv).
 
-.. _Annotate_images:
+After setting up the ``Annotator`` instance, you can interactively annotate a sample of your images using:
 
-Annotate your images
-----------------------
-
-.. note:: Run these commands in a Jupyter notebook (or other IDE), ensuring you are in your `mr_py38` python environment.
+.. code-block:: python
 
-To prepare your annotations, you must specify a ``userID``, ``annotation_tasks_file`` (i.e. the ``annotation_task.yaml``), tell MapReader which ``task`` you'd like to run and which  ``annotation_set`` you would like to run on.
+    annotator.annotate()
 
-.. todo:: Give big list of different options here
-.. todo:: Explain that things don't autosave
+Context
+~~~~~~~
 
-e.g. following our 'rail_space' example from earlier:
+To help with annotating, you can set the annotation interface to show a context image using ``show_context=True``.
+This creates a panel of patches in the annotation interface, highlighting your patch in the middle of its surrounding immediate images.
```

Comment:
Change to "This creates a panel of patches in the annotation interface, highlighting your patch in the middle of its immediately surrounding area."

Question @rwood-97 - this is always 1 patch distance away, right? If so, can we say that explicitly above?

Reply:
Will add docs for this - you can add ``surrounding`` kwarg to set it to more than 1.